### ISSUE TITLE
Get all parent elements in API v0.7

### ISSUE THREAD

**Issue by zbycz** [openstreetmap-ng/openstreetmap-ng#75]
The GET item endpoint should have an option to fetch all parent relations (and for nodes also all parent ways).

It scarcely make sense to edit some element without the knowledge of its parents. It would be very helpful for editors and other usage. And currently in v0.6 it is only possible with an additional request for [`/node/123/ways`](https://wiki.openstreetmap.org/wiki/API_v0.6#Ways_for_node:_GET_/api/0.6/node/#id/ways) or [`node/123/relations`](https://wiki.openstreetmap.org/wiki/API_v0.6#Relations_for_element:_GET_/api/0.6/[node|way|relation]/#id/relations).

via https://github.com/Zaczero/openstreetmap-ng/blob/4ae2d74/ANNOUNCEMENT.md#new-api-07

btw, is there some issue to track v0.7 progress or discussion? I would be glad to subscribe :)

**Comment by Zaczero** [comment]
So what you're proposing is that there should be a single endpoint (or an option) to fetch all of an element's parents? Because the issue currently is that you have to perform several requests, increasing your code's complexity: checking for types, and deciding whether to fetch /ways or /relations.

I have created an "API 0.7" label on GitHub, which should make it easier to discover these kinds of issues. Currently, feature parity is the top priority, so you won't find much about API 0.7 yet (I have not moved my notes to GitHub yet). However, if you have ideas or suggestions today, feel free to post them.

**Comment by zbycz** [comment]
Yes my propsal is exactly what you wrote. Thanks. I will watch the label 👍
